tests/fixtures/whats-cooking-2021-02-10.mbox

```diff
@@ -405,7 +405,7 @@ Release tarballs are available at:
  Will merge to 'next' after waiting for a few more days for comments.
 
 
-* cw/pack-vs-bigfilethreashold (2021-02-09) 2 commits
+* cw/pack-vs-bigfilethreshold (2021-02-09) 2 commits
```

Comment:
Technically, this is incorrect, as I copied this email verbatim. Changing it would no longer be verbatim.

In practice, it probably does not matter much, so I'm fine with this change anyway.